### PR TITLE
SONAR-7425 Stop computing technical debt measures per rule

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: sonar-db/src/main/java/org/sonar/db/measure/MeasureDto.java [hunk 1]
```diff
@@ -190,13 +189,11 @@ public MeasureDto setSnapshotId(Long snapshotId) {
...
-  @CheckForNull
-  public Integer getRuleId() {
-    return ruleId;
-  }
-
+  /**
+   * @deprecated in 5.5. Does nothing. Kept for compatibility with developer cockpit plugin, version 1.10
+   */
+  @Deprecated
   public MeasureDto setRuleId(@Nullable Integer ruleId) {
```
Comment (SonarTech) [inline sonar-db/src/main/java/org/sonar/db/measure/MeasureDto.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by teryk** [comment]
Jira tickets should be renamed. We're removing any measure per rule, not just technical debt. Even if in practice, it's equivalent

**Comment by julienlancelot** [comment]
You're right, JIRA ticket has been renamed

**Comment by teryk** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* INFO 1 info

Watch the comments in this conversation to review them.

**Pull request merged**